### ISSUE TITLE
Clarify factory patch/wavetable/skin/data licensing for XT2.0

### ISSUE THREAD

**Issue by selenologist** [surge-synthesizer/surge#6741]
GPLv3 is a poor fit for the type of material in our factory data. There are a lot of difficult questions that arise, like, is a modified factory patch a derivative work? If a factory patch is included in your DAW project files, is your DAW project now GPLv3? etc

Considering many patches will be culled anyway in XT2.0, this presents a good opportunity to re-license this data (with the permission of all IP owners) to a license that is designed for something other than code, that clarifies matters like these.

@baconpaul has suggested [CC-BY-SA 4.0](https://creativecommons.org/licenses/by-sa/4.0/).

**Comment by selenologist** [comment]
Making a note that patches in the Templates folder should be licensed more permissively than BY-SA (possibly CC0), as the great majority of patches derive from e.g. Init Saw, and Templates should be expected to not contain ShareAlike elements.

**Comment by trynsta** [comment]
Please correct me if I am wrong but is it not like licensing patches under CC BY-SA is forcing producer who used these presets to publish his track under the same license? Is beat made with CC BY-SA presets a derivative work?

**Comment by baconpaul** [comment]
Yeah CC0 is better and what we went with for the template patches we re-licensed explicitly in XT 1.x
